### ISSUE TITLE
Human readable export format

### ISSUE THREAD

**Issue by giorgiopogliani** [verbb/formie#326]
**What are you trying to do?**
I am trying to export submissions in a human readable format

**What's your proposed solution?**
I found myself in the need of doing the following in order to create a better CSV export. 

```php
// foreach $fieldValues as $value

if ($value instanceof craft\elements\db\EntryQuery) {
    return array_reduce($value->all(), function ($acc, $input) {
        return $acc . $input->title;
    }, "");
}

if ($value instanceof SingleOptionFieldData) {
    return $value->value;
}

if (is_string($value)) {
    return $value;
}

if (is_bool($value)) {
    return $value ? 'YES' : 'NO';
}

// .... and others 
```

I sent this export with an automatic email but the same problem applies to the export inside control panel. 

**Additional context**
In details for example, something like this { "number":"0123456789", "country":null, "hasCountryCode":false } can be hard to understand to non-tech people.

**Comment by engram-design** [comment]
What version of Formie are you on? 

Your example with the phone export should've been fixed in https://github.com/verbb/formie/commit/d6111792616d4ff3ad750143225b64fbda1cc252

Although I agree that some fields can be improved, like element fields and checkboxes which I've just included in the craft-3 branch for the next release. See https://github.com/verbb/formie/commit/451e2f551870fd9d9438a84a4c9f990733d2c1da and https://github.com/verbb/formie/commit/2a928a901569ba8a553054330821154e146ee95f

I've also added a new event for you to modify the values used by exports - https://github.com/verbb/formie/commit/26576ced1dbe3e6ff92d8f6479a87b41d4150282

**Comment by giorgiopogliani** [comment]
Yes, actually I am not on the latest, I am on `1.3.7`. I had problems upgrading but I will try again and check the exports. I will also checkout the `EVENT_MODIFY_FIELD_EXPORT `, Thanks!

**Comment by engram-design** [comment]
Fixed in [1.3.16](https://github.com/verbb/formie/releases/tag/1.3.16)
